package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@scaleway/serverless-functions",
   "version": "1.1.0",
   "description": "",
-  "main": "framework/dist/local/index.js",
-  "types": "framework/dist/local/index.d.ts",
```

Comment:
@cyclimse and @Bemilie this is what really killed me, this path was not correct, so I was getting a load of "Could not find package" errors even though it was installed :(

Reply:
Arf, so sorry for it :pray:

Reply:
Np, not your fault as it clearly worked with it being wrong :smile:. Just wanted to flag in case you have similar issues in future and think "Ah, I remember something about a path in the `package.json`"